### DISCUSSION TITLE
How to know if the killed player was blind?

### DISCUSSION THREAD

**Discussion by pedrofornaza** [saul/demofile#212] [Questions] [answered]
I will have to do it manually, using "player_blinded" event? 

It returns me the blind time but how i can keep track of this time "passing" ?

**Comment by saul** [comment]
There are some properties on the player like m_flFlashDuration (time blind in seconds). You can grab the current game time off the DemoFile class.

**Comment by pedrofornaza** [comment]
Nice! I will try it later and close the issue after. Is that ok?

**Comment by saul** [comment, marked as answer]
Sure that's fine. To tell if a player's blind you need to do something similar to:

1. When player_blinded is fired, keep track of the following `unblindTime = demo.currentTime + player.getProp('DT_CSPlayer', 'm_flFlashDuration')`.
2. To determine if a player is blind, do: `unblindTime < demo.currentTime`

Let me know if you've got any issues.

**Comment by pedrofornaza** [comment]
Tested and working. Thanks!
